PanModal/Controller/PanModalPresentationController.swift

```diff
@@ -234,12 +246,57 @@ open class PanModalPresentationController: UIPresentationController {
             }
         })
     }
-
+    
+    // MARK: - Actions
+    
+    /**
+     Update long form height and short form height when the keyboard is showing/scrolling and scroll to correct position above the keyboard if keyboard is showing
+     */
+    @objc func keyboardWillShow(notification: Notification) {
+        
+        //if the presented view is already at max height and is scrollable then scroll to above the keyboard when keyboard is shown if needed
+        
+        if let keyboardSize = (notification.userInfo?  [UIResponder.keyboardFrameEndUserInfoKey] as? NSValue)?.cgRectValue {
```

Comment:
That's a bit crazy.. why not get it as a GSRect in the first place..
`let kbRect = userInfo[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect,`